src/Codec/Xlsx/Parser.hs

```diff
@@ -136,69 +149,60 @@ extractCellValue _ "b" "0" = [CellBool False]
 extractCellValue _ _ _ = []
 
 -- | Get xml cursor from the specified file inside the zip archive.
-xmlCursor :: Zip.Archive -> FilePath -> Maybe Cursor
-xmlCursor ar fname = parse <$> Zip.findEntryByPath fname ar
+xmlCursor :: Zip.Archive -> FilePath -> Either ParseError (Maybe Cursor)
+xmlCursor ar fname = maybe (Right Nothing) parse $ Zip.findEntryByPath fname ar
```

Comment:
and for this `hush` could be used - http://hackage.haskell.org/package/errors-2.0.0/docs/Control-Error-Util.html#v:hush